x-pack/plugin/searchable-snapshots/src/main/java/org/elasticsearch/index/store/SearchableSnapshotDirectory.java

```diff
@@ -50,95 +42,24 @@
  * shard files and what it stored in the snapshot the {@link BlobStoreIndexShardSnapshot} is used to map a physical file name as expected by
  * Lucene with the one (or the ones) corresponding blob(s) in the snapshot.
  */
-public class SearchableSnapshotDirectory extends BaseDirectory {
-
-    private final BlobStoreIndexShardSnapshot snapshot;
-    private final BlobContainer blobContainer;
+public class SearchableSnapshotDirectory extends BaseSearchableSnapshotDirectory {
```

Comment:
This is only used if `index.store.snapshot.cache.enabled` is false. Maybe we can remove that setting and this whole class?

Reply:
We discussed this in another channel and agreed to defer this to a followup.